<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>889</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dungeon and Hex Move Tracker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>630</y>
      <width>221</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Tracking</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>201</width>
       <height>205</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="cbGroundType"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chFreshSnow">
        <property name="text">
         <string>Fresh snow cover</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Size of creature:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbTrackSize"/>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Num Creatures</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tbNumCreatures"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Days old</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="tbTrackAge"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Hours of rain</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="tbRainHours"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>540</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>LIghting</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pbTorch">
          <property name="text">
           <string>Torch</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pbCandle">
          <property name="text">
           <string>Candle</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbLamp">
        <property name="text">
         <string>Lamp Oil</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Move/Wait</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>19</y>
       <width>201</width>
       <height>184</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pbHexMove">
        <property name="text">
         <string>Hex Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDungeonMove">
        <property name="text">
         <string>Dungeon Move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbUrbanMove">
        <property name="text">
         <string>Urban Move</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QPushButton" name="AddMin30">
          <property name="text">
           <string>+30 Min</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="AddHr1">
          <property name="text">
           <string>+1 Hr</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="AddMin1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>+1 Min</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="AddMin10">
          <property name="text">
           <string>+10 Min</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="AddHr4">
          <property name="text">
           <string>+4 Hr</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="AddHr8">
          <property name="text">
           <string>+8 Hr</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>221</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Factors</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>29</y>
       <width>201</width>
       <height>269</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QComboBox" name="cbMoveMode"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbTerrain"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbRoad"/>
      </item>
      <item>
       <widget class="QCheckBox" name="chGiantTerrain">
        <property name="text">
         <string>Giant Terrain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chPoorVis">
        <property name="text">
         <string>Poor visibility (fog, dark)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chColdClimate">
        <property name="text">
         <string>Cold or hot climate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbSnowCover"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbConditions"/>
      </item>
      <item>
       <widget class="QComboBox" name="cbActivities"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Stats</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>27</y>
       <width>201</width>
       <height>244</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Base Move:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="MoveRate">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Squares Per Turn:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="SPT"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Hex Per Watch:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="HPW"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current Time:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="TimeDisplay">
        <property name="text">
         <string>09:00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Miles Per Watch:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="tbMilesPerWatch"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Forage DC</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="tbForage"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Progress in Hex</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="tbHexProgress"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Miles Left in Hex</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="tbMilesLeft"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>201</width>
       <height>311</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="MessageOut"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>430</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Rest</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>20</y>
       <width>201</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="pbShortRest">
        <property name="text">
         <string>Short Rest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbLongRest">
        <property name="text">
         <string>Long Rest</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbForceRest">
        <property name="text">
         <string>Force Rested</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>290</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Navigation</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>201</width>
       <height>118</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QCheckBox" name="cbNavTools">
        <property name="text">
         <string>Has Navigation Tools</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chHexNearExit">
        <property name="text">
         <string>Exiting Hex Near Side</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Navigation Mod</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="tbNavMod"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Nav DC</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="tbNavDc"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
